src/Serialization/main.jl

```diff
@@ -398,13 +398,44 @@ julia> meta = metadata(author_orcid="0000-0000-0000-0042", name="42", descriptio
 Oscar.MetaData("0000-0000-0000-0042", "42", "The meaning of life, the universe and everything")
 
 julia> save("/tmp/fourtitwo.json", 42; metadata=meta);
+ERROR: ArgumentError: '/tmp/fourtitwo.json' exists. `force=true` is required to remove '/tmp/fourtitwo.json' before copying.
```

Comment:
Error?

Reply:
Thanks. I guess this is what happens if you use `doctest = :fix` without heeding the warnings...